reference/scalability_guide/mongo_cursor_timeout_exception.rst

```diff
@@ -0,0 +1,21 @@
+How to get rid of the MongoCursorTimeoutException
+-------------------------------------------------
+
+In the PIM we always iterate over cursor and paginate them to avoid big queries on the mongo database. But for some low level operation which are not paginable you could encounter the following exception:
+
+.. code-block:: yaml
+
+    [MongoCursorTimeoutException] localhost:27017: Read timed out after reading 0 bytes, waited for 30.000000 seconds
+
+In this case, there is no real solution except increasing the timeout limit by doing the following:
+
+.. code-block:: yaml
+
+    # ./app/config/config_mongodb.yml
```

Comment:
app/config/config.yml

people work with standard edition
don't forget it
config_mongodb.yml does not exist